### ISSUE TITLE
[iOS] Improve TextBox support for screen readers

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno#12684]
I'm trying to improve the screen reader support for the UnoApplicationTemplate. The behavior for `TextBox` on iOS is not up to standards for accessibility. 

Right now in the app template, when focusing a `TextBox`, VoiceOver reads it like so:

- If `TextBox` is empty:
  - `PlaceholderText`
- If `TextBox` is filled:
  - `PlaceholderText`

In comparison, when you use the form to create a new contact on the iOS built-in app Contacts, VoiceOver reads it that way:

- If `TextBox` is empty:
  - `Label`
  - `PlaceholderText`
- If `TextBox` is filled:
  - `Label`
  - Text input by user

What doesn't work is:
- Using `AutomationProperties.Name` in xaml
- Using `AutomationProperties.HelpText` in xaml

Do you have ideas on what would be the best way to support VoiceOver for `TextBox`?

_Originally posted by @doum-vigeant-nventive in https://github.com/unoplatform/uno/discussions/12679_

**Comment by MartinZikmund** [comment]
We will be reworking accessibility support for iOS on Skia targets soon, closing this native issue for now
